deps.edn

```diff
@@ -328,42 +329,24 @@
   :oss-dev
   {}
 
-  ;; Generate BE documentation with
-  ;; clojure -M:marginalia
-  :marginalia
-  {:extra-deps
-   {com.github.tsmacdonald/marginalia {:mvn/version "0.9.2"}}
-   :main-opts ["-m" "marginalia.main" "-n" "Metabase" "-d" "backend-docs" "-D"
-               "The simplest, fastest way to get business intelligence and analytics to everyone in your company 😋"
-               "dev" "src" "enterprise/backend/src"]}
-
-  ;; Generate a single file's docs with
-  ;;   clojure -M:marginalia/one src/metabase/some/namespace.clj
-  ;; which creates `backend-docs/metabase.some.namespace.html`
-  :marginalia/one
-  {:extra-deps
-   {com.github.tsmacdonald/marginalia {:mvn/version "0.9.2"}}
-   :main-opts ["-m" "marginalia.main" "-m" "-n" "Metabase" "-d" "backend-docs" "-D"
-               "The simplest, fastest way to get business intelligence and analytics to everyone in your company 😋"]}
```

Comment:
We do it [a different way](https://github.com/metabase-dev-docs/metabase-dev-docs.github.io/) now